drivers/classic/src/get_global_param.c

```diff
@@ -92,11 +78,14 @@ get_global_param(FILE *gp)
             else if (strcasecmp("NODES", optstr) == 0) {
                 sscanf(cmdstr, "%*s %zu", &options.Nnode);
             }
-            else if (strcasecmp("TIME_STEP", optstr) == 0) {
-                sscanf(cmdstr, "%*s %u", &global_param.dt);
+            else if (strcasecmp("MODEL_STEPS_PER_DAY", optstr) == 0) {
+                sscanf(cmdstr, "%*s %zu", &global_param.model_steps_per_day);
```

Comment:
Yes, that is how I currently have it setup.  I will raise an error if `TIME_STEP` or `SNOW_STEP` is used and direct the user to add the `MODEL_STEPS_PER_DAY` and `SNOW_STEPS_PER_DAY`.